### ISSUE TITLE
Annoying quotation marks at input when tampering JWT

### ISSUE THREAD

**Issue by dhondta** [ticarpi/jwt_tool#26]
Hi @ticarpi 

While trying to tamper JWT values, I figured out that, when providing an input, it adds annoying quotation marks, resulting in an error for integers (e.g. `ValueError: invalid literal for int() with base 10: '"12345"'`) and making invalid strings.

**Comment by ticarpi** [comment]
Looks like an old error snuck in.
Should be fixed now in v2.0.1
Thanks so much for reporting.

**Comment by ticarpi** [comment]
Ah, looks like I was fixing an unrelated "quoting" issue.  
@dhondta you can avoid this issue by using double-quotes (") instead of single quotes (') in Tamper mode.  
Have reopened this issue for visibility in case other people experience the same.  
I'll update the docs (and maybe the UI) when I get the chance.

**Comment by dhondta** [comment]
Hi @ticarpi 

Here is what I get (using your example JWT from the help message) : 

```sh
$ python jwt_tool.py -T eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbiI6InRpY2FycGkifQ.bsSwqj2c2uI9n7-ajmi3ixVGhPUiY7jO9SUn9dm15Po

        \   \        \         \          \                    \ 
   \__   |   |  \     |\__    __| \__    __|                    |
         |   |   \    |      |          |       \         \     |
         |        \   |      |          |    __  \     __  \    |
  \      |      _     |      |          |   |     |   |     |   |
   |     |     / \    |      |          |   |     |   |     |   |
\        |    /   \   |      |          |\        |\        |   |
 \______/ \__/     \__|   \__|      \__| \______/  \______/ \__|
  Version 2.0                 \______|             @ticarpi      

Original JWT: 


====================================================================
This option allows you to tamper with the header, contents and 
signature of the JWT.
====================================================================

Token header values:
[1] typ = "JWT"
[2] alg = "HS256"
[3] *ADD A VALUE*
[4] *DELETE A VALUE*
[0] Continue to next step

Please select a field number:
(or 0 to Continue)
> 3
Please enter new Key and hit ENTER
> jku
Please enter new value for jku and hit ENTER
> http://127.0.0.1/jwks.json
[1] typ = "JWT"
[2] alg = "HS256"
[3] jku = ""http://127.0.0.1/jwks.json""
[4] *ADD A VALUE*
[5] *DELETE A VALUE*
[0] Continue to next step

```

As you can see, my new field "`jku`" has its value enclosed in quotes, which will be reflected in the JWT and therefore causing an error when the server checking for the token will try to open the link, invalid because of the quotes.

Another example, using the quotes as you suggested : 

```sh
Token header values:
[1] typ = "JWT"
[2] alg = "HS256"
[3] *ADD A VALUE*
[4] *DELETE A VALUE*
[0] Continue to next step

Please select a field number:
(or 0 to Continue)
> 2

Current value of alg is: HS256
Please enter new value and hit ENTER
> 
[1] typ = "JWT"
[2] alg = """"
[3] *ADD A VALUE*
[4] *DELETE A VALUE*
[0] Continue to next step

Please select a field number:
(or 0 to Continue)
> 2

Current value of alg is: ""
Please enter new value and hit ENTER
> "TEST"
[1] typ = "JWT"
[2] alg = ""\"TEST\"""
[3] *ADD A VALUE*
[4] *DELETE A VALUE*
[0] Continue to next step

Please select a field number:
(or 0 to Continue)
> 
```

The tampering mode is really convenient when it comes to modifying and signing a token. But this issue with the quotes prevents it from working. Could you fix this, please ?

**Comment by ticarpi** [comment]
Yes, you should find this was all fixed in v2.0.1
Update your version and check it out.
I copy-pasted your exact commands above and they worked perfectly for me.
Let me know the errors if it's still not working :)

**Comment by dhondta** [comment]
Arf, my bad ! I did not figure out that my last `git pull` failed because of local changes. I re-cloned the repo and now I have effectively the latest version.
My apologies for the inconvenience and thanks for this awesome tool !
